internal/tools/go.mod

```
@@ -0,0 +1,437 @@
+module github.com/observiq/bindplane-agent/internal/tools
+
+go 1.22
+
+toolchain go1.22.3
```

```suggestion
go 1.21.9
```
Should match the main `go.mod`